shell/platform/darwin/macos/framework/Source/FlutterResizableBackingStoreProvider.h

```diff
@@ -0,0 +1,58 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#import <Cocoa/Cocoa.h>
+#import <Metal/Metal.h>
+#import <QuartzCore/CAMetalLayer.h>
+
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterBackingStoreDescriptor.h"
+#import "flutter/shell/platform/darwin/macos/framework/Source/FlutterResizeSynchronizer.h"
+
+/**
+ * Represents a buffer that can be resized.
+ */
+@protocol FlutterResizableBackingStoreProvider <FlutterResizeSynchronizerDelegate>
+
+/**
+ * Notify of the required backing store size updates. Called during window resize.
+ */
+- (void)updateBackingStoreIfNecessaryForSize:(CGSize)size;
```

Comment:
Is there a reason not to just call this something like `onBackingStoreResized:`?